common/src/main/scala/explore/syntax/ui/package.scala

```diff
@@ -1,80 +1,55 @@
+  def clearableNoPadding(using ev: ExternalValue[EV], ev3: Eq[A]) = {
+    val newClazz: UndefOr[Css] =
+      input.clazz.fold(ExploreStyles.ClearableInputPaddingReset)(
+        _ |+| ExploreStyles.ClearableInputPaddingReset
+      )
+    input.copy(icon = clearInputIcon[EV, A](input.value), clazz = newClazz)
   }
 
-  implicit class FormInputEVOps[EV[_], A, B](val input: FormInputEV[EV, Option[A]]) extends AnyVal {
-    def clearable(implicit ev: ExternalValue[EV], ev3: Eq[A]) =
-      input.copy(icon = clearInputIcon[EV, A](input.value))
+extension [EV[_], A, B](input: InputWithUnits[EV, Option[A]])
+  def clearable(using ev: ExternalValue[EV], ev3: Eq[A]) =
+    input.copy(icon = clearInputIcon[EV, A](input.value))
 
-    // When an icon is added to a FormInputEV, SUI adds extra padding on the right to make
-    // space for the icon. However, with some layouts this can cause resizing issues, so this
-    // method removes that extra padding. See `clearInputIcon` for more details.
-    def clearableNoPadding(implicit ev: ExternalValue[EV], ev3: Eq[A]) = {
-      val newClazz: UndefOr[Css] =
-        input.clazz.fold(ExploreStyles.ClearableInputPaddingReset)(
-          _ |+| ExploreStyles.ClearableInputPaddingReset
-        )
-      input.copy(icon = clearInputIcon[EV, A](input.value), clazz = newClazz)
-    }
+  // When an icon is added to a FormInputEV, SUI adds extra padding on the right to make
+  // space for the icon. However, with some layouts this can cause resizing issues, so this
+  // method removes that extra padding. See `clearInputIcon` for more details.
+  def clearableNoPadding(using ev: ExternalValue[EV], ev3: Eq[A]) = {
+    val newClazz = input.clazz |+| ExploreStyles.ClearableInputPaddingReset
+    input.copy(icon = clearInputIcon[EV, A](input.value), clazz = newClazz)
   }
 
-  implicit class InputWithUnitsOps[EV[_], A, B](val input: InputWithUnits[EV, Option[A]])
-      extends AnyVal {
-    def clearable(implicit ev: ExternalValue[EV], ev3: Eq[A]) =
-      input.copy(icon = clearInputIcon[EV, A](input.value))
+extension [A](c: js.UndefOr[A => Callback])
+  def toJs: js.UndefOr[js.Function1[A, Unit]] = c.map(x => (a: A) => x(a).runNow())
 
-    // When an icon is added to a FormInputEV, SUI adds extra padding on the right to make
-    // space for the icon. However, with some layouts this can cause resizing issues, so this
-    // method removes that extra padding. See `clearInputIcon` for more details.
-    def clearableNoPadding(implicit ev: ExternalValue[EV], ev3: Eq[A]) = {
-      val newClazz = input.clazz |+| ExploreStyles.ClearableInputPaddingReset
-      input.copy(icon = clearInputIcon[EV, A](input.value), clazz = newClazz)
-    }
-  }
-
-  extension [A](c: js.UndefOr[A => Callback])
-    def toJs: js.UndefOr[js.Function1[A, Unit]] = c.map(x => (a: A) => x(a).runNow())
-
-}
+extension (c: Callback.type) def pprintln[T](a: T): Callback = Callback(_root_.pprint.pprintln(a))
```

Comment:
I think I'll move this to lucuma-ui

Reply:
Nice.

Can we have a `Logger.pdebug` too, please?

Reply:
We have this too, is it in use?
https://github.com/gemini-hlsw/explore/blob/8f7d99556e53adc49f073e53184b258a7d26667a/common/src/main/scala/explore/utils/pprinter.scala

Reply:
I had completely forgotten about that.  It doesn't seem to be in use.